examples/miscellaneous/plot_combining_naive_bayes.py

```diff
@@ -0,0 +1,125 @@
+"""
+===================================================
+Combining Naive Bayes Estimators using ColumnwiseNB
+===================================================
+
+.. currentmodule:: sklearn
+
+This example shows how to use :class:`~naive_bayes.ColumnwiseNB`
+meta-estimator to construct a naive Bayes model from base naive Bayes
+estimators. The resulting model is applied to a dataset with a mixture of
+discrete and continuous features.
+
+We consider the titanic dataset, in which:
+
+- numerical (continous) features "age" and "fare" are handled by
+  :class:`~naive_bayes.GaussianNB`;
+- categorical (discrete) features "embarked", "sex", and "pclass" are handled
+  by :class:`~naive_bayes.CategoricalNB`.
+"""
+
+# Author: Andrey V. Melnik <andrey.melnik.maths@gmail.com>
+#         (based on a related work by Pedro Morales <part.morales@gmail.com>)
```

Comment:
```suggestion
#         Pedro Morales <part.morales@gmail.com>
```

Reply:
@partmor I hope you are fine with the attribution/credit, as I am largely reusing your work from PR [#11197](https://github.com/scikit-learn/scikit-learn/pull/11197).

Reply:
sure! I'm happy it helped you!